erpnext/crm/doctype/investor/investor_dashboard.py

```diff
@@ -3,13 +3,16 @@
 
 def get_data():
 	return {
-		'fieldname': 'investor_name',
+		'heatmap': True,
+		'heatmap_message': _('This is based on the transaction of this Investor'),
+		'fieldname': 'name',
```

Comment:
`name` is the primary key, you'll be trying to match against the `name` of the linking document, which will never be true.

Keep this as `investor`